classes/task/queue_to_process_pending_merge_requests.php

```diff
@@ -0,0 +1,72 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+namespace tool_mergeusers\task;
+use \tool_mergeusers\merge_request;
+/**
+ * Version information
+ *
+ * @package     tool
+ * @subpackage  mergeusers
+ * @author      Nicola Vallinoto, Liguria Digitale
+ * @author      Jordi Pujol-Ahulló, SREd, Universitat Rovira i Virgili
+ * @license     http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+class queue_to_process_pending_merge_requests extends \core\task\scheduled_task {
+    public function get_name() {
+        return get_string('queue_to_process_pending_merge_requests', 'tool_mergeusers');
+    }
+    /**
+     * Run task for getting merge requests and adding them to adhoc task list.
+     * 
+     */
+    public function execute() {
+        mtrace("Task queue_to_process_pending_merge_requests started.");
```

Comment:
you can use the `self::class` (I think it can be used like this) to print the full classpath name of this class. So you do not need to worry to update this log if the class name, someday, is modified.